File: test/fixtures/csharp/get_user_agent.cs
using System.Net.Http;

HttpClient client = new HttpClient();

HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Get, "http://localhost:28139/vc/moviesmagic?p=5&pub=testmovie&tkn=817263812");

request.Headers.Add("x-msisdn", "XXXXXXXXXXXXX");
request.Headers.Add("user-agent", "Mozilla Android6.1");

HttpResponseMessage response = await client.SendAsync(request);
response.EnsureSuccessStatusCode();
string responseBody = await response.Content.ReadAsStringAsync();

File: test/fixtures/java/get_user_agent.java
import java.io.IOException;
import java.net.URI;
import java.net.http.HttpClient;
import java.net.http.HttpRequest;
import java.net.http.HttpResponse;

HttpClient client = HttpClient.newBuilder()
    .followRedirects(HttpClient.Redirect.NORMAL)
    .build();

HttpRequest request = HttpRequest.newBuilder()
    .uri(URI.create("http://localhost:28139/vc/moviesmagic?p=5&pub=testmovie&tkn=817263812"))
    .GET()
    .setHeader("x-msisdn", "XXXXXXXXXXXXX")
    .setHeader("user-agent", "Mozilla Android6.1")
    .build();

HttpResponse<String> response = client.send(request, HttpResponse.BodyHandlers.ofString());
